### PR TITLE
feat: add connect-to-nx-cloud command

### PR DESCRIPTION
**Pull request by MaxKless**

### CODE REVIEW

File: libs/vscode/nx-help-and-feedback-view/src/lib/nx-help-and-feedback-provider.ts [hunk 1]
```diff
@@ -42,4 +62,50 @@ export class NxHelpAndFeedbackProvider extends AbstractTreeProvider<NxHelpAndFee
       ([title, link, icon]) => new NxHelpAndFeedbackTreeItem(title, link, icon)
     );
   }
+
+  async isConnectedToCloud(): Promise<boolean> {
+    const nxJson = (await readJsonFile(
+      join(getWorkspacePath(), 'nx.json')
+    )) as NxJsonConfiguration;
```
Comment (Cammisuli) [inline libs/vscode/nx-help-and-feedback-view/src/lib/nx-help-and-feedback-provider.ts:69]: Use the `nxWorkspace` function to get this information. Right now the nx specific data isn't exposed, but you can get it from here:
https://github.com/nrwl/nx-console/blob/21c5cc5246c6e46c0e959d4346ef93105972a30b/libs/shared/workspace/src/lib/workspace.ts#L95-L100

and return it somewhere here:
https://github.com/nrwl/nx-console/blob/21c5cc5246c6e46c0e959d4346ef93105972a30b/libs/shared/workspace/src/lib/workspace.ts#L103-L120

File: libs/vscode/nx-help-and-feedback-view/src/lib/nx-help-and-feedback-provider.ts [hunk 2]
```diff
@@ -42,4 +62,50 @@ export class NxHelpAndFeedbackProvider extends AbstractTreeProvider<NxHelpAndFee
...
+            { cwd: getWorkspacePath() }
+          );
+
+          commandProcess.stdout.setEncoding('utf8');
+
+          commandProcess.stdout.on('data', (data) => {
+            if (data.includes('I want faster builds')) {
+              commandProcess.stdin.write('yes\r\n', 'utf8');
+            }
+          });
```
Comment (Cammisuli) [inline libs/vscode/nx-help-and-feedback-view/src/lib/nx-help-and-feedback-provider.ts:104]: I'm fairly certain we can do a `--no-interaction` or some other kind of flag to automatically accept the defaults.

File: libs/vscode/nx-help-and-feedback-view/src/lib/nx-help-and-feedback-provider.ts (hunk 1 above)
Comment (MaxKless) [reply libs/vscode/nx-help-and-feedback-view/src/lib/nx-help-and-feedback-provider.ts:69]: Seems like the info I need is already returned in the `workspace` property...

File: libs/vscode/nx-help-and-feedback-view/src/lib/nx-help-and-feedback-provider.ts (hunk 2 above)
Comment (MaxKless) [reply libs/vscode/nx-help-and-feedback-view/src/lib/nx-help-and-feedback-provider.ts:104]: I couldn't find anything like that...

File: libs/vscode/nx-help-and-feedback-view/src/lib/nx-help-and-feedback-provider.ts [hunk 3]
```diff
@@ -42,4 +111,84 @@ export class NxHelpAndFeedbackProvider extends AbstractTreeProvider<NxHelpAndFee
...
+
+    window.withProgress(
+      {
+        location: ProgressLocation.Notification,
+        title: 'Connecting you to Nx Cloud...',
+      },
+      () => {
+        return new Promise((resolve) => {
+          try {
+            if (lt(coerce(nxVersion) ?? '', '12.0.0')) {
```
Comment (Cammisuli) [inline libs/vscode/nx-help-and-feedback-view/src/lib/nx-help-and-feedback-provider.ts:146]: This is just a note to self: move the version parser in a util function to use for other packages (like lerna)

**Pull request merged**